project/src/main/java/com/google/sps/servlets/QueryServlet.java

```diff
@@ -42,10 +44,12 @@
   static final int MAX_NUM_PLACES_TO_RECOMMEND = 3;
   private PlacesFetcher fetcher;
 
+    // The entry point for a Google GEO API request.
+  private static final GeoApiContext CONTEXT = GeoContext.getGeoApiContext();
```

Comment:
Now that you have a singleton factory, you don't really need to hold it as a static member of your own. 
You can just have `new PlacesFetcher(GeoContext.getGeoApiContext());`

If you need many, just store it as a temporary variable in the `init()` function.

Reply:
Totaly right